pom.xml

```diff
@@ -14,48 +14,56 @@
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <jstl.version>3.0.0</jstl.version>
+        <jakarta.version>6.0.0</jakarta.version>
+        <postgres.version>42.6.0</postgres.version>
+        <gson.version>2.10.1</gson.version>
+        <logback.version>1.4.7</logback.version>
+        <glassfish.version>3.0.1</glassfish.version>
     </properties>
+
     <dependencies>
         <dependency>
             <groupId>jakarta.servlet</groupId>
             <artifactId>jakarta.servlet-api</artifactId>
-            <version>6.0.0</version>
+            <version>${jakarta.version}</version>
```

Comment:
Better set full name and do not use dot `.` like: `jakarta-servlet-api-version`.

Reply:
fixed